pkg/network/tracer/tracer_linux_test.go

```diff
@@ -2154,3 +2155,37 @@ func (s *TracerSuite) TestOffsetGuessIPv6DisabledCentOS() {
 	// fail if tracer cannot start
 	_ = setupTracer(t, cfg)
 }
+
+func (s *TracerSuite) TestPESisDisabledForUnsupportedKernels() {
+	t := s.T()
+	cfg := testConfig()
+	cfg.EnableProcessEventMonitoring = true
+	kv, err := ebpfkernel.NewKernelVersion()
+	kv4150 := kernel.VersionCode(4, 15, 0)
+	require.NoError(t, err)
+	if kv.Code >= kv4150 || kv.IsRH8Kernel() {
+		t.Skip("This test should only be run on kernels < 4.15.0 and RH7")
```

Comment:
Don't we expect that the tracer actually starts on those kernels?  The behavior of the tracer in this revision is that the tracer will skip the PEDS initialization if it detects that the tracer is on a too-old tracer version.